### ISSUE TITLE
Column support for currency kind of values

### ISSUE THREAD

**Issue by kartik1000** [phpmyadmin/phpmyadmin#15169]
### Is your feature request related to a problem? Please describe.

Few days back while working in some hackathon, I realised that nowadays we often use databases, for say storing amount of certain items, something like what Amazon does but seeing the amount of different items as simple integral values is a bit frustrating as neither does one get to understand if it is amount of something or in which currency type has the amount been stored in the database.

### Describe the solution you'd like

Personally, I would love to have a clear distinction between normal integer type columns and currency type columns either using different column headers and adding currency sign say '$' against each entry. Actually other ideas are also welcomed on how we can make them more distinctive.

### Possible Solution

One of the solutions or enhancements to it can be made by adding a transformation plugin as suggested to me by @williamdes 

### Additional context

- phpMyAdmin version: 4.8.4

**Comment by saurass** [comment]
Yeah that would be great, @kartik1000 , PMA already supports some transformation. It would be great if currency transformation is implemented.

We can even implement something like currency conversion in real time. We can use some APIs to convert a currency to other. It will enhance user experience.

**Comment by kartik1000** [comment]
Yes, using an API for conversion definitely seems a great idea to me too @saurass. It will definitely be useful for users using database for market related activities.

**Comment by williamdes** [comment]
@kartik1000 I am not sure that depending on an api is a good idea :confused: 
@ibennetch what do you think about that idea?

**Comment by kartik1000** [comment]
@williamdes, is it because if the api host make some change into it then our converter/transformation might crash?

**Comment by saurass** [comment]
> @williamdes, is it because if the api host make some change into it then our converter/transformation might crash?

Yeah maybe this is one of the reason.

This will surely enhance user experience, but also this isn't much of scope for phpMyAdmin project.

**Comment by williamdes** [comment]
@kartik1000 Yes
- Data leak
- Api closing down
- Api deprecation
- Api changing to paid plan
- Api not supporting currencies and then users posting issues to PMA
Lots of things like the points above

**Comment by kartik1000** [comment]
@williamdes, I think transformation plugin idea would be better then and i think we can wait for @ibennetch review for this one
